QuizApp/QuizApp/UI/User/UserViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+import SnapKit
+
+class UserViewController: UIViewController {
+
+    private let gradient = CAGradientLayer()
+
+    private var viewModel: UserViewModel!
+    private var stackView: UIStackView!
```

Comment:
I want you to practice constraints. Can you please rewrite this without UIStackView?